vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -201,9 +245,93 @@ void Unblinded::PrepareTokens(
     return;
   }
 
-  // TODO(https://github.com/brave/brave-browser/issues/8887):
-  // we should reserve this tokens and add step STEP_RESERVE
-  PreparePublishers(token_list, std::move(contribution), types, callback);
+  const std::string contribution_id = contribution->contribution_id;
+  const std::string contribution_string =
+      braveledger_bind_util::FromContributionToString(std::move(contribution));
+
+  auto reserved_callback = std::bind(
+      &Unblinded::OnMarkUnblindedTokensAsReserved,
+      this,
+      _1,
+      token_list,
+      contribution_string,
```

Comment:
Possible to `std::move` these two?

Reply:
I fixed it for the token list. Are you suggesting also for the string?

Reply:
Up to you - depends on how big the string is I suppose.

Reply:
OK, will leave as-is for now as I don't think this string grows very large. Will keep it in mind though, as I have definitely been erring more towards favoring its use with containers.